gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemXAttrsIntegrationTest.java

```diff
@@ -59,13 +61,20 @@ public final class GoogleHadoopFileSystemXAttrsIntegrationTest {
 
   private String bucketName;
 
+  @Parameterized.Parameter public boolean testStorageClientImpl;
```

Comment:
Taking into account that we would want to test all our clients, and in future there will be more than 2 of the, should we use exiting `ClientType` enum for parametrizing tests, as boolean looks a little weird and harder maintain going forward?

Reply:
Updated.